ovos_tts_plugin_piper/util.py

```
@@ -0,0 +1,713 @@
+        "kinda": "kind of",
+        "let's": "let us",
+        "ma'am": "madam",
+        "may've": "may have",
+        "mayn't": "may not",
+        "mightn't've": "might not have",
+        "mustn't've": "must not have",
+        "needn't've": "need not have",
+        "ol'": "old",
+        "oughtn't've": "ought not have",
+        "sha'n't": "shall not",
+        "shan't": "shall not",
```

Ruff (0.12.2)

Dictionary key literal `"shan't"` repeated

Remove repeated key literal `"shan't"`

(F601)